README.md

```diff
@@ -1,27 +1,28 @@
 # july-challenge-tbd
 
-_Pending_: one-liner about CRUD app
+A web app that acts as a directory in finding employees in their areas of expertise and team involvment.
```

Comment:
@martinmags 
I think the sentence is read a little more easily this way:
A web app that acts as a directory to find employees based on their areas of expertise and team involvement.